justfile

```diff
@@ -14,6 +16,7 @@ log := "warn"
 
 export JUST_LOG := log
 
+# ltest is installed via `cargo install cargo-limit`
```

Comment:
this is implicitly documented in the `install-dev-deps` target as all other development dependencies.

Reply:
Maybe a comment at the top of the file, or the readme about `install-dev-deps` then?

Reply:
i won't be less confused when you tell me things about Windows. ;-)

Reply:
That would be _even better_. There are no real descriptive docs there on how to get started.